jwql/website/apps/jwql/data_containers.py

```diff
@@ -460,13 +460,18 @@ def get_expstart(instrument, rootname):
         The exposure start time of the observation (in MJD).
     """
 
-    file_set_name = '_'.join(rootname.split('_')[:-1])
+    if '-seg' in rootname:
```

Comment:
Is `-seg` in some kind of engineering or ground test data?

Reply:
They appear to be some files related to time series observations: https://innerspace.stsci.edu/pages/viewpage.action?pageId=94092600#SummaryofAssociationFileNamingConvention/Design-TSOfiles

Reply:
Time series observations that create a data volume above some limit are split into multiple "segment" files. "seg" refers to the segment number. So you can have a single exposure split up into, eg:
jw88888002001_01101_00002-seg001_nrca5_uncal.fits
jw88888002001_01101_00002-seg002_nrca5_uncal.fits
jw88888002001_01101_00002-seg003_nrca5_uncal.fits

Reply:
Thanks @bhilbert4!